### PR TITLE
integration test: add agent loglevel test

### PR DESCRIPTION
**Pull request by dougsland**
Adding test for the LogLevel option in the configuration.
Related #416 

Signed-off-by: Douglas Schilling Landgraf <dougsland@redhat.com>

### CODE REVIEW

**Comment by dougsland** [comment]
**Testing locally**: 

**OS**:
```
# cat /etc/redhat-release
CentOS Stream release 9
```

**Running**:
```
# echo "tier: 1" >> tests/tier0/bluechi-agent-loglevel/main.fmf
# tmt run test --filter tier:1

/var/tmp/tmt/run-001

/plans/tier0
    discover
        how: fmf
        directory: /root/bluechi/tests
        filter: tier:1
        summary: 1 test selected
    provision
        how: local
        multihost name: default-0
        arch: x86_64
        distro: CentOS Stream 9
        summary: 1 guest provisioned
    prepare
        queued push task #1: push on default-0

        push task #1: push on default-0

        queued prepare task #1: Set containers setup on default-0

        prepare task #1: Set containers setup on default-0
        how: shell
        name: Set containers setup
        overview: 1 script found

        queued pull task #1: pull on default-0

        pull task #1: pull on default-0

        summary: 1 preparation applied
    execute
        queued execute task #1: default-0 on default-0

        execute task #1: default-0 on default-0
        how: tmt
        progress:
        summary: 1 test executed
    report
        how: junit
        output: /var/tmp/tmt/run-001/plans/tier0/report/default-0/junit.xml
        summary: 1 test passed
    finish

        summary: 0 tasks completed

total: 1 test passed
```

File: tests/tests/tier0/bluechi-agent-loglevel/test_agent_loglevel.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+    _, output = nodes[NODE_WITH_NUMBERS_ONLY_IN_LOGLEVEL].exec_run('systemctl status bluechi-agent')
+    output_systemd = str(output)
+    if "fail" in output_systemd.lower():
+        raise Exception(
+                f"{NODE_WITH_NUMBERS_ONLY_IN_LOGLEVEL} bluechi-agent should "
+                "NOT failed during the start of the service")
+
+
+@pytest.mark.timeout(45)
```
Comment (engelmi) [inline tests/tests/tier0/bluechi-agent-loglevel/test_agent_loglevel.py]: Nit: I think 15s should be fine.

File: tests/tests/tier0/bluechi-agent-loglevel/test_agent_loglevel.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+        raise Exception(
+                f"{NODE_WITH_NOT_VALID_VALUE} bluechi-agent should "
+                "NOT failed during the start of the service")
+
+    _, output = nodes[NODE_WITH_NUMBERS_ONLY_IN_LOGLEVEL].exec_run('systemctl status bluechi-agent')
+    output_systemd = str(output)
+    if "fail" in output_systemd.lower():
+        raise Exception(
+                f"{NODE_WITH_NUMBERS_ONLY_IN_LOGLEVEL} bluechi-agent should "
+                "NOT failed during the start of the service")
```
Comment (engelmi) [inline tests/tests/tier0/bluechi-agent-loglevel/test_agent_loglevel.py:42]: Optional:
You could also check if the nodes are shown as on-/offline by using the python bindings (see example [here](https://github.com/containers/bluechi/tree/main/tests/tests/tier0/monitor-open-close)).

**Review by engelmi — APPROVED** [review]
A minor nit, otherwise LGTM

File: tests/tests/tier0/bluechi-agent-loglevel/test_agent_loglevel.py (hunk 2 above)
Comment (dougsland) [reply tests/tests/tier0/bluechi-agent-loglevel/test_agent_loglevel.py:42]: Sure, as I have the approval. I will move forward with the patch what we have now just chaning the timeout. The next one I will use bindings, thanks for the review.

File: tests/tests/tier0/bluechi-agent-loglevel/test_agent_loglevel.py (hunk 1 above)
Comment (dougsland) [reply tests/tests/tier0/bluechi-agent-loglevel/test_agent_loglevel.py]: sure

**Pull request merged**